Doc/library/datetime.rst

```diff
@@ -65,7 +65,7 @@ understand and to work with, at the cost of ignoring some aspects of reality.
 
 For applications requiring aware objects, :class:`.datetime` and :class:`.time`
 objects have an optional time zone information attribute, :attr:`!tzinfo`, that
-can be set to an instance of a subclass of the abstract :class:`tzinfo` class.
+can be set to an instance of a subclass of the abstract :class:`!tzinfo` class.
```

Comment:
@erlend-aasland I removed this link and kept the next one in the same paragraph because I think the next occurrence is more relevant.